gateway-service/src/main/resources/application.yml

```diff
@@ -257,6 +257,7 @@ eureka:
                 service:
                     title: API Gateway
                     description: API Gateway service to route requests to services registered in the API Mediation Layer and provides an API for mainframe security.
+                    supportClientCertForwarding: true
```

Comment:
not sure if this should be true as default

Reply:
I hope, the API Gateway will support proxying of client certs in the future as well :D